packages/story-editor/src/elements/video/controls.js

```diff
@@ -45,7 +44,7 @@ function VideoControls({
       isTransforming={isTransforming}
       elementRef={elementRef}
       element={element}
-      videoRef={videoRef}
```

Comment:
Removed `videoRef` since it does not receive anything and passes `undefined` to `playpausebutton`.